### ISSUE TITLE
IntelliSense Broken in IntelliJ

### ISSUE THREAD

**Issue by HankG** [yshrsmz/twitter4kt#22]
I'm attempting to work on the development of the library with IntelliJ 2020.2 with Kotlin 1.4 and it seems like the IntelliSense and dependencies lookup aren't working. Are there some plugins I should be using with this hierarchical Gradle configuration?

**Comment by yshrsmz** [comment]
Hmm, that's strange.
I'm using IntelliJ IDEA 2020.2.3 and Kotlin plugin 1.4.10-release-IJ2020.1, and it works fine for me.

This issue could be related.
https://youtrack.jetbrains.com/issue/KT-41141#focus=Comments-27-4348035.0-0
There could be an empty JDK11 in the project structure, and you need to remove it.

**Comment by HankG** [comment]
Yeah everything is JDK 11 on that system. I'll look into that. Thanks!
